core/docs/spark-qualification-tool.md

```diff
@@ -32,50 +20,70 @@ applicable environments.  Here are the cluster information for the ETL benchmark
+   - [On active jobs using a Spark Listener](#running-the-qualification-tool-using-a-spark-listener)
+- [Understanding the Qualification Tool Output](#understanding-the-qualification-tool-output)
+   - [Application Report](#application-report)
+   - [Stages Report](#stages-report)
+   - [Execs Report](#execs-report)
+   - [MLFunctions Report](#mlfunctions-report)
+   - [MLFunctions Total Duration Report](#mlfunctions-total-duration-report)
+   - [HTML Report](#html-report)
+- [Example Reports and File Output](#example-reports-and-file-output)
+- [Compiling the Tools Jar from Source](#compiling-the-tools-jar-from-source)
+- [S3 File Path Support and Configuration](#s3-file-path-support-and-configuration)
+- [Tools Jar Maven Dependency](#qualification-tool-maven-dependency-snippet)
 
-This document covers below topics:
 
-* TOC
-{:toc}
 
-## How to use the Qualification tool
+## Running the Qualification Tool
 
-The Qualification tool can be run in three different ways. One is to run it as a standalone tool on the
-Spark event logs after the application(s) have run, the second is to be integrated into a running Spark
-application using explicit API calls, and the third is to install a Spark listener which can output
-results on a per SQL query basis.
+The Qualification tool can be run three different ways:
+1. A standalone application running on Spark event logs after the applications are done
+2. Integrated into a running Spark application using explicit API calls
+3. Installing a Spark listener to output results on a per SQL query basis
 
-In running the qualification tool standalone on Spark event logs, the tool can be run as a user tool command
-via a [pip package](https://pypi.org/project/spark-rapids-user-tools/) for CSP environments (Google Dataproc,
-AWS EMR, Databricks AWS) or as a java application for other environments.
+Standalone mode can be run as a java application or using the `spark_rapids_user_tools` application that is available as the `spark-rapids-user-tools` [pip package](https://pypi.org/project/spark-rapids-user-tools/).
 
-## Running the Qualification tool standalone for CSP environments on Spark event logs
-### User Tools Prerequisites and Setup for CSP environments
+## Running the Standalone Qualification Tool for CSP Environments on Spark Event Logs
+### User Tools Prerequisites and Setup for CSP Environments
 
-* [Dataproc](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-dataproc.md)
-* [EMR](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-aws-emr.md)
-* [Databricks AWS](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-databricks-aws.md)
+* [Dataproc](../../user_tools/docs/user-tools-dataproc.md)
+* [EMR](../../user_tools/docs/user-tools-aws-emr.md)
+* [Databricks AWS](../../user_tools/docs/user-tools-databricks-aws.md)
```

Comment:
we cannot use relative URLs here because:

1. those markdown files are synced to plugin RAPIDS-repo which don't have user-tools documentation.
2. we want to use absolute path pointing to the "main" branch because this is the stable branch representing the latest release.